agent/functional_tests/tests/generated/simpletests_unix/simpletests_generated_unix_test.go

```diff
@@ -775,10 +775,22 @@ func TestNetworkLink(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Could not register task definition: %v", err)
 	}
-	testTasks, err := agent.StartMultipleTasks(t, td, 1)
-	if err != nil {
-		t.Fatalf("Could not start task: %v", err)
+	var testTasks []*TestTask
+	if "" == "true" {
```

Comment:
This looks a bit suspect?

Reply:
Its auto generated -- i believe the goal was to skip a compiler check. Basically, some tests are duplicated with/without awsvpc mode. 

I don't think we've checked in a full run of "make gogenerate" in a while, so that's why it is showing up suddenly in this review.